topostats/roughness.py

```diff
@@ -0,0 +1,17 @@
+"""Contains functions pertaining to the roughness of AFM images."""
+
+import numpy as np
+
+
+def roughness_rms(image: np.ndarray) -> float:
+    """Calculate the rms roughness of a heightmap image.
```

Comment:
Is RMS here the [Root Mean Square](https://en.wikipedia.org/wiki/Root_mean_square)? Might be useful to spell this out in the docstring.